replay(options): point out how to opt-in for text if safe (#8531)
getsentry/sentry-docs: src/platform-includes/replay/privacy-configuration/javascript.mdx

> For a static website, free of personal identifiable or other type of private data.

Reviewer: ```suggestion
If you're working on a static website that's free of personal identifiable or other type of private data,
```

> You can easily opt-out out of the default text masking and image blocking by configuring the `maskAllText` and `blockAllMedia` configuration options respectively:

Reviewer: ```suggestion
you can opt out of the default text masking and image blocking by configuring the `maskAllText` and `blockAllMedia` configuration options respectively:
```

```javascript
new Sentry.Replay({
  maskAllText: false,
  blockAllMedia: false,
});
```

> The following is the complete list of options that can be used in `new Replay({})`:

Reviewer: ```suggestion
The following is a complete list of options that can be used in `new Replay({})`:
```

> For a static website, free of personal identifiable or other type of private data.

Reviewer 2: This recommendation placed on two docs pages feels quite aggressive and far from the privacy-first approach. :/ 

I wonder if we should still mark _inputs_ as masked in this default configuration. 

Also, I think we should talk about PII server-side scrubbing in place as it's additional insurance

Author: > This recommendation placed on two docs pages feels quite aggressive and far from the privacy-first approach. :/

We're still privacy-first. We need to be clear that there's the option and too often people come out confused that this is an option.

I could move it from the include to the _Configuration_ only page. Leaving the Privacy one without it? 
But since we have it in both I feel this fits well.

> I wonder if we should still mark inputs as masked in this default configuration.

You mean point out in this paragraph that user text will continue to be masked even with that? 

> Also, I think we should talk about PII server-side scrubbing in place as it's additional insurance

We talk about it somewhere else on the docs, do you mean also including it here?

> You can easily opt-in to text and media with:

Reviewer 3: I would invert this as the user is choosing to opt out of our defaults

```suggestion
You can easily opt-out out of the default text masking and image blocking by configuring the `maskAllText` and `blockAllMedia` configuration options respectively:
```

```javascript

> new Sentry.Replay({
>       maskAllText: false,
>       blockAllMedia: false,
>     }),
>   },
> });

Reviewer 3: ```suggestion
new Sentry.Replay({
  maskAllText: false,
  blockAllMedia: false,
});
```

> ```

Author: @jas-kas this?

````suggestion
```
Note that user input will continued to be masked.
````
